pkg/transform/kubernetes/kubernetes.go

```diff
@@ -1,122 +1,39 @@
-package pkg
+package kubernetes
 
 import (
-	"fmt"
-	"io/ioutil"
-	"os"
-
 	"github.com/davecgh/go-spew/spew"
-	"github.com/ghodss/yaml"
 	"github.com/pkg/errors"
 	"k8s.io/client-go/pkg/api"
+
 	"k8s.io/client-go/pkg/api/resource"
 	"k8s.io/client-go/pkg/runtime"
 	"k8s.io/client-go/pkg/util/intstr"
 
 	log "github.com/Sirupsen/logrus"
-	api_v1 "k8s.io/client-go/pkg/api/v1"
-	ext_v1beta1 "k8s.io/client-go/pkg/apis/extensions/v1beta1"
 
-	// install api
```

Comment:
also keep this comment with more info maybe so that people know why those errors of not installed comes?